bigtable-hbase-1.x-parent/bigtable-hbase-1.x-hadoop/pom.xml

```diff
@@ -118,7 +118,6 @@ limitations under the License.
               <goal>shade</goal>
             </goals>
             <configuration>
```

Comment:
does this also stop creating an artifact/publishing the test jar? If no, could we add it as part of this?

Reply:
not sure, I'll come back to this after the dependencies have settled